src/date-range-picker/embedded.tsx

```diff
@@ -0,0 +1,129 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useRef } from 'react';
+import Calendar, { DayIndex } from './calendar';
+import InternalBox from '../box/internal';
+import SpaceBetween from '../space-between/index.js';
+
+import RelativeRangePicker from './relative-range';
+import ModeSwitcher from './mode-switcher';
+import { useDateRangePicker } from './use-date-range-picker';
+import { DateRangePickerBaseProps } from './interfaces';
+import { fireNonCancelableEvent } from '../internal/events/index.js';
+import { DateRangePickerProps, Focusable } from './interfaces';
+
+export interface DateRangePickerDropdownProps
+  extends Pick<
```

Comment:
Since the interfaces are now split into BaseProps and Props -- it is still required to have a Pick? If a few props to be excluded that can probably be simplified with Omit.

Reply:
We can definitely remove it